src/Material/Textfield.elm

```diff
@@ -349,8 +351,19 @@ view lift model options =
         Nothing -> []
 
 
-    listeners =
-      config.listeners
+    -- Internal events that require special handling because user provided events
+    -- might overlap with these
+    internal =
+      [[ ("focus", Decoder.succeed (lift Focus))
```

Comment:
Maybe add a variant of `on` to Dispatch for the very common case of not wanting to decode.  Then you'd be able to write:

    [[ ("focus", Dispatch.on1 (lift Focus))

(Not sure about the name `on1`.)